src/groupInfo/datastore/groups.js

```diff
@@ -41,6 +41,7 @@ export default {
       return Object.values(state.entries).map(getters.enrich)
     },
     mine: (state, getters) => getters.all.filter(e => isMyGroup(e)).sort(applicationsFirstThenSortByName),
+    isMemberGroups: (state, getters) => getters.all.filter(e => isMemberGroup(e)).sort(applicationsFirstThenSortByName),
```

Comment:
Minor: you can simplify this:

```js
.filter(e => isMemberGroup(e))
```

to this:

```js
.filter(isMemberGroup)
```

Because `e => isMemberGroup(e)` is just a function that takes one argument (`e`) and calls one function (`isMemberGroup`), that's also the same as just the reference to the function `isMemberGroup` itself...

Reply:
Got it! I copied the line above, `(e => isMyGroup(e))`, so I changed both getters now to this simplified reference to the function